mercure/Caddyfile.platform_sh

```diff
@@ -16,6 +16,9 @@ log {
 	}
 }
 
+#cors
+header Access-Control-Allow-Credentials true
```

Comment:
Is it required? Probably we could explain it better in the comment about why and when it can be used?